source/codegen/metadata/nidcpower/CHANGES.md

```diff
@@ -1,7 +1,35 @@
-# Changes in metadata from the nimi-python metadata
+# Changes in metadata from the base metadata
+
+## config.py
+
+The following metadata was added:
+- 'c_function_prefix': 'niDCPower_'
+- 'service_class_prefix': 'NiDCPower'
+- 'java_package': 'com.ni.grpc.dcpower'
+- 'csharp_namespace': 'NationalInstruments.Grpc.DCPower'
+
+The following function name was capitalised:
+- 'close_function': 'Close'
+
+## functions.py
+
+The following functions were tagged with 'init_method': True, to ensure their generated service handler registers the new session with the session_repository:
+- InitializeWithIndependentChannels
+- InitializeWithChannels
+- InitExtCal
+
+The following functions were tagged with their corresponding c_names:
+- reset
+- self_test
+- revision_query
+- ConfigureOVP
+- error_message
+- error_query
 
```

Comment:
@ni-pkhare , you had once made changes to MeasureMultiple API which was using source code mechanism know? Can you update that in changes.md file?

Reply:
Since there is another [task ](https://ni.visualstudio.com/DevCentral/_workitems/edit/1426043) for implementing MeasureMultiple API in which we will be adding CustomCode tag, we can update Changes.md file in that task only for that API. I have mentioned this in that task. So resolving the comment.